### PR TITLE
Store name in future wallet

### PR DESCRIPTION
**Pull request by Marik-D**
# Summary
A description of the changes proposed in the pull request.

Fixes #number

## Checklist
- [ ] Change is small and easy to review (please split big changes into multiple PRs)
- [ ] Change is consistent with architecture
- [ ] Change is consistent with test architecture
- [ ] Change is consistent with naming conventions
- [ ] New code is covered with tests
- [ ] Tests related to old code are updated
- [ ] Documentation is up to date with changes

### CODE REVIEW

File: universal-login-sdk/tsconfig.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "allowJs": false,
     "declaration": true,
     "outDir": "dist",
+    "sourceMap": true,
```
Comment (Marik-D) [inline universal-login-sdk/tsconfig.json]: remo this

Comment (Marik-D) [inline universal-login-sdk/tsconfig.json]:
```suggestion
```

File: universal-login-sdk/test/integration/core/WalletService.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ describe('INT: WalletService', () => {
   const deployedWallet: DeployedWallet = new DeployedWallet('0x123', 'justyna.nylogin.eth', '0x29F3EDEE0AD3ABF8E2699402E0E28CD6492C9BE7EAAB00D732A791C33552F779', sdk);
   const applicationWallet = deployedWallet.asApplicationWallet;
   const futureWallet: FutureWallet = {
+    name: 'justyna.nylogin.eth',
```
Comment (Marik-D) [inline universal-login-sdk/test/integration/core/WalletService.ts]:
```suggestion
    name: 'justyna.mylogin.eth',
```

File: universal-login-sdk/test/integration/core/WalletService.ts [hunk 3]
```diff
@@ -14,6 +14,7 @@ describe('INT: WalletService', () => {
   const deployedWallet: DeployedWallet = new DeployedWallet('0x123', 'justyna.nylogin.eth', '0x29F3EDEE0AD3ABF8E2699402E0E28CD6492C9BE7EAAB00D732A791C33552F779', sdk);
```
Comment (Marik-D) [inline universal-login-sdk/test/integration/core/WalletService.ts]:
```suggestion
  const deployedWallet: DeployedWallet = new DeployedWallet('0x123', 'justyna.mylogin.eth', '0x29F3EDEE0AD3ABF8E2699402E0E28CD6492C9BE7EAAB00D732A791C33552F779', sdk);
```

**Comment by marekkirejczyk** [comment]
We agreed to move the name to WalletService to keep FutureWallet immutable.

**Pull request closed without merging**